### DISCUSSION TITLE
Discussion about implimentation

### DISCUSSION THREAD

**Discussion by peepo5** [wassup-/bybit-rs#3] [General]
Hi there, I am glad that you have found an interest in my project :) it is a personal development project and I intend to use it for myself while sharing it with others. I have some things to add before adding another exchange, and i am accepting feature requests with open arms :))

My question though: is this repo going to be maintained? By the way I do not mind helping with it as well.

**Comment by wassup-** [comment]
I do plan on maintaining this repository for the near future. I also plan on keeping this repository as focused as possible, which means that ideally it should just be about providing the actual API/WebSocket connection with bybit. Any other functionality (such as managing open orders etc.) should be contained in the repository of the consumer of this library.

However as I said before I'm open to any suggestions, feature requests, ... ! 
So do not hesitate to share your ideas here :)

**Comment by wassup-** [comment]
I'm also open to helping you out with termcrypt. I'm still in the process of learning Rust, and I see this as a great way to broaden my knowledge and skills. I work best in a ticket/issue based manner, so feel free to create issues and assign them to me if you feel like I can be of assistance.

**Reply by peepo5** [reply]
:+1: :D  I also started learning rust about a month ago so we are on the same page. What I would recommend as a suggestion would be adding a list using the markdown format `- [x]` or `- [ ]` as a checklist of completed endpoints & features.

**Reply by wassup-** [reply]
I have updated the README to include a checklist of completed endpoints & features.
Making this checklist made me realise I still have work to do on this library, I'll try to do some of it this weekend.
It would be great if you could indicate which features should be prioritised!

I have also updated the code as to remove most of the unwraps.

**Reply by peepo5** [reply]
Sounds great. It is a good idea to removw unwraps everywhere where possible so errors can be contained and handled. I will give you some ideas of endpoints to prioritize tomorrow. Good work.
